site/app/templates/LateDaysTablePlugin.twig

```diff
@@ -32,22 +32,20 @@
     </tr>
     </thead>
     <tbody id="late_day_table">
-    {% set num_status = 0 %}
     {% for gradeable in student_gradeables %}
         {% if g_id != NULL and gradeable.getId() == g_id %}
             {% set class = "class=blue-background" %}
             {% set id = "id = curr_status" %}
         {% endif %}
         <tr>
-            <td {{ class }} style="padding:5px; border:thin solid black">{{gradeable.getName()}}</td>
-            <td {{ class }} align="center" style="padding:5px; border:thin solid black">{{gradeable.getDueDate()|date('m/d/Y')}}</td>
-            <td {{ class }} align="center" style="padding:5px; border:thin solid black">{{gradeable.getAllowedLateDays()}}</td>
-            <td {{ class }} align="center" style="padding:5px; border:thin solid black">{{(gradeable.getLateDays() != 0) ? gradeable.getLateDays() : ""}}</td>
-            <td {{ class }} align="center" style="padding:5px; border:thin solid black">{{(gradeable.getLateDayExceptions() != 0) ? gradeable.getLateDayExceptions() : ""}}</td>
-            <td {{ class }} {{ id }} align="center" style="padding:5px; border:thin solid black">{{attribute(status_array, num_status)}}</td>
-            <td {{ class }} align="center" style="padding:5px; border:thin solid black">{{(attribute(late_charged_array, num_status) != 0 ) ? attribute(late_charged_array, num_status) : ""}}</td>
+            <td {{ class }} style="padding:5px; border:thin solid black">={{ gradeable.getName() }}</td>
```

Comment:
The = sign should be removed, leading to 
![weird equal signs after names](https://user-images.githubusercontent.com/11095297/42459682-2ddc0492-836a-11e8-89ec-cecdcfe4f7cf.JPG)